CHANGELOG.md

```diff
@@ -93,6 +105,11 @@ v1.10.0
 ### Fixed
 - Included `resolutions` option to `package.json` to ensure version of `gift` installed is up-to-date and doesn't break the build.
 
+<<<<<<< HEAD
+=======
```

Comment:
Need to tidy up these conflict markers, and it looks like the few lines below this have been added in the wrong place?